x-pack/plugins/alerting/server/mocks.ts

```diff
@@ -64,6 +64,17 @@ const createAlertFactoryMock = {
 
     return mock as unknown as AlertInstanceMock<InstanceState, InstanceContext>;
   },
+  done: <
+    InstanceState extends AlertInstanceState = AlertInstanceState,
+    InstanceContext extends AlertInstanceContext = AlertInstanceContext,
+    ActionGroupIds extends string = string
+  >() => {
+    const mock = {
+      getRecoveredAlerts: jest.fn(),
+    };
+    mock.getRecoveredAlerts.mockReturnValue([]);
```

Comment:
**optional nit:**
```suggestion
    const mock = {
      getRecoveredAlerts: jest.fn().mockReturnValue([]),
    };
```

Reply:
Updated in https://github.com/elastic/kibana/pull/124972/commits/afe0c45743cd5a14e4bff3b5523846d02179058a